src/frontend/js/custom-leaflet.js

```diff
@@ -32,13 +32,13 @@ function printMapAndItinerary() {
     printContainer.remove();
 }
 
-function getWaypoints(routeId) {
+function getRouteWaypoints(routeId) {
     var waypoints = [];
     // Reset current waypoints
     control.setWaypoints(waypoints);
 
     // Ajax call to get waypoint according route
-    $.get( "../getDailyOrders/?route="+routeId, function(data ) {
+    $.get( "../getDailyOrders/?route="+routeId+"&mode=euclidean", function(data ) {
```

Comment:
I implemented the view to reload a saved route, but did not add this functionality in the template, since I saw last week that you were working on a new "route organizer UI" and wanted to talk to you beforehand.